catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/controller/ScriptController.java

```diff
@@ -59,6 +62,30 @@ public Script copy(Script scriptToCopy, Project dstProject, Scene dstScene, Spri
 			throws IOException, CloneNotSupportedException {
 
 		Script script = scriptToCopy.clone();
+
+		if (script instanceof UserDefinedScript) {
```

Comment:
Please rethink structure of this function, too many lines "outsourcing" to own function.